src/sagemaker/clarify.py

```diff
@@ -25,9 +25,11 @@
 
 import tempfile
 from abc import ABC, abstractmethod
-from typing import List, Union, Dict
```

Comment:
Currently we don't support clarify processor integrated directly with Pipeline. Thus no PipelineVariable annotations are added to this `clarify.py`